hazelcast/src/main/java/com/hazelcast/cp/internal/RaftService.java

```diff
@@ -1457,6 +1458,19 @@ public int getCPMemberPriority() {
         return cpMemberPriority;
     }
 
+    public void wipeDestroyedObjects() {
```

Comment:
Blocking -- wasn't clear if there was any reason to make this any more complex.

?